arangod/Graph/Enumerators/WeightedTwoSidedEnumerator.cpp

```diff
@@ -824,14 +838,6 @@ typename WeightedTwoSidedEnumerator<QueueType, PathStoreType, ProviderType,
                                     PathValidator>::BallSearchLocation
 WeightedTwoSidedEnumerator<QueueType, PathStoreType, ProviderType,
                            PathValidator>::getBallToContinueSearch() {
-  if (!_leftInitialFetch) {
```

Comment:
nit: it also looks like that the `getBallToContinueSearch()` method doesn't mutate the object and thus could be made `const`.